pkg/util/ec2/imds_helpers.go

```diff
@@ -24,13 +24,13 @@ var (
 	imdsNetworkMacs = "/network/interfaces/macs"
 )
 
-// IMDSv2Usage is an enum to determine how to interact with the IMDSv2 option
-type IMDSv2Usage int
+// IMDSVersion is an enum to determine how to interact with the IMDSv2 option
+type IMDSVersion int
```

Comment:
`IMDSVersion` doesn't need to be exported.

Reply:
good catch, done